components/dash-core-components/src/fragments/Graph.react.js

```diff
@@ -374,6 +374,27 @@ class PlotlyGraph extends Component {
         });
     }
 
+    getStyle() {
+        const {responsive} = this.props;
+        let {style} = this.props;
+
+        // When there is no forced responsive style, return the original style property
+        if (!responsive) {
+            return style;
+        }
+
+        // Otherwise, if the height is not set, we make the graph size equal to the parent one
+        if (!style) {
+            style = {};
+        }
+
+        if (!style.height) {
+            style.height = '100%';
```

Comment:
```suggestion
            return Object.assign({height: '100%'}, style);
```

We shouldn't be modifying `style` in place, could cause issues if the user turns off `responsive`, or reads `style` back out in another callback.